### PR TITLE
feat(index): add home sections 

### PR DESCRIPTION
**Pull request by MrTimscampi**
Adds support for the following home sections:

* resume
* resumeaudio
* nextup
* latest

Some current sections have been omitted voluntarily for now:

* librarytiles
  Makes sense later on, but not a priority
* librarybuttons
  Would be implemented with librarytiles and rely on the same component
* activerecordings
  Not a priority, annoying to test
* onnow/livetv
  Needs a visual redesign, since we don't want the buttons there, only programs

image

### CODE REVIEW

**Comment by ThibaultNocchi** [comment]
Maybe add some x-margins to the cards?

**Comment by MrTimscampi** [comment]
> Maybe add some x-margins to the cards?
> 
> Edit: I think a section should be a v-row, and all cards should be wrapped in v-col.
> 
> Edit 2: It seems each section is already in its v-row

The card component needs a fully custom component at some point. The one we use now is just a placeholder to visualize info.

**Comment by ThibaultNocchi** [comment]
No problem then. If I understand correctly, there'll be a need for a component which adapts to the type of content (music, show, movie...)?

File: components/HomeSection.vue [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    <div class="d-flex flex-wrap">
+      <card
+        v-for="item in items"
+        :key="item.Id"
+        :to="`/item/${item.Id}`"
+        class="card mt-5"
+        :item="item"
+      />
+    </div>
+  </v-col>
```
Comment (Camc314) [inline components/HomeSection.vue:16]: Probably better to use v-slide-group and v-slide-item

**Comment by MrTimscampi** [comment]
> No problem then. If I understand correctly, there'll be a need for a component which adapts to the type of content (music, show, movie...)?

Yeah, our card component needs to have a few specific sizes for various items types (not a straight image like now), have an overlay and some other things.

The cardBuilder from jf-web has what we need, for reference, but it's obviously done in a very messy way.

**Comment by MrTimscampi** [comment]
`v-slide-group` works for now, but we'll likely want to find something with a lot more features down the line.

**Review by Camc314 — APPROVED** [review]

**Pull request merged**